src/twisted/internet/_sslverify.py

```diff
@@ -1525,11 +1604,27 @@ def __init__(self,
         self._mode = SSL.MODE_RELEASE_BUFFERS
 
         if method is None:
-            # If no method is specified set things up so that TLSv1.0 and newer
-            # will be supported.
             self.method = SSL.SSLv23_METHOD
-            self._options |= SSL.OP_NO_SSLv3
+
+            if minimumTLSVersion is None:
+                minimumTLSVersion = TLSVersion.TLSv1_0
```

Comment:
This value should probably take account of `maximumTLSVersion`: if you have set `maximumTLSVersion=SSLv3` (because you hate yourself) then this default value will fail, even though we have enough information to DTRT.